blockchain.go

```diff
@@ -1011,6 +1012,48 @@ func (b *Blockchain) commitBlock() (*flowgo.Block, error) {
 	return block, nil
 }
 
+func (b *Blockchain) GetAccountStorage(address sdk.Address) (*AccountStorage, error) {
+	program := programs.NewEmptyPrograms()
+
+	env, err := fvm.NewTransactionEnvironment(
+		b.vmCtx,
+		b.vm,
+		state.NewStateHolder(
+			state.NewState(
+				b.pendingBlock.ledgerView,
+				meter.NewMeter(math.MaxUint64, math.MaxUint64),
+				state.WithMaxKeySizeAllowed(b.vmCtx.MaxStateKeySize),
+				state.WithMaxValueSizeAllowed(b.vmCtx.MaxStateValueSize),
+				state.WithMaxInteractionSizeAllowed(b.vmCtx.MaxStateInteractionSize),
```

Comment:
this one `WithMaxInteractionSizeAllowed`?

Reply:
yes. `WithMaxValueSizeAllowed` and `WithMaxKeySizeAllowed` dont matter here, because the are the max size of a key or a value. `WithMaxInteractionSizeAllowed` matters because it gets incremented as you read stuff from storage, so eventually it will error.